templates/intro-jobs/README.ipynb

```diff
@@ -97,6 +97,24 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "## Submitting a Job programmatically\n",
+    "\n",
+    "In the above section, you submitted a Job from a workspace. By default, Jobs submitted from workspaces inherit the dependencies and compute config of the workspace.\n",
+    "\n",
+    "You can also submit jobs from other machines, using the Anyscale CLI. \n",
+    "\n",
+    "Copy ``main.py`` to an empty folder on your **laptop**, and then run the following on your **laptop** to try this out:\n",
+    "\n",
+    "```bash\n",
+    "# Make sure we have anyscale CLI installed.\n",
+    "$ pip install -U anyscale\n",
+    "\n",
+    "# Note: will package all contents of your current working dir into the job deps.\n",
+    "$ anyscale job submit --wait -- python main.py\n",
```

Comment:
this has to pass `--working-dir=.`, it's currently only defaulted when in workspaces. i don't remember why we decided on that

Reply:
Oh, shall we change that? Seems like a good default to have.

Reply:
I filed https://anyscale1.atlassian.net/browse/OA-725 to unblock this PR.

Reply:
The problem with setting it by default is we'd also need to have a `--no-working-dir` option, else there's no way to disable uploading. I'd be OK with it if we add that option as well.